res/shaders/customLCD.shader/manifest.ini

```diff
@@ -0,0 +1,65 @@
+[shader]
+name=Custom LCD
+author=endrift, Dominus Iniquitatis, EddyHg80
+description=Configurable LCD emulation.
+passes=4
+
+[pass.0]
+fragmentShader=customLCD-brightness.fs
+blend=1
+width=-1
+height=-1
+
+[pass.0.uniform.brightness]
+type=float
+default=1.1
+readableName=Brightness
+
+[pass.1]
+fragmentShader=customLCD-subPixels.fs
+blend=1
+width=-3
+height=-3
+
+[pass.1.uniform.subPixelDimming]
+type=float
+default=0.2
+readableName=Sub pixel dimming
+
+[pass.2]
+fragmentShader=customLCD-pixelBounds.fs
+blend=1
+width=-6
+height=-6
+
+[pass.2.uniform.boundBrightness]
+type=float
+default=0.9
+readableName=Pixel bound brightness
+
+[pass.3]
+fragmentShader=ags001-light.fs
+blend=1
+width=-6
+height=-6
```

Comment:
Since these two passes operate at the same size, I might recommend merging them into a single pass.

Reply:
The reason it's separated is because I wanted to keep the settings in two different tabs.
Tell me if you agree or if I should merge them anyway.